src/ripple/app/ledger/LedgerConsensus.h

```diff
@@ -56,16 +56,16 @@ class LedgerConsensus : public Traits
 
     virtual LgrID_t getLCL () = 0;
 
-    virtual void gotMap (TxSet_t const& map) = 0;
+    virtual void gotMap (Time_t const& now, TxSet_t const& map) = 0;
```

Comment:
I'm not sure we need that customization point, as long as the implementation gets its time from `abstract_clock<NetClock>` then unit tests can provide their own clock.